akka-persistence/src/test/scala/akka/persistence/PersistentActorBoundedStashingSpec.scala

```diff
@@ -66,8 +66,11 @@ class SteppingInMemPersistentActorBoundedStashingSpec(strategyConfig: String)
     system.eventStream.publish(Mute(EventFilter.warning(pattern = ".*received dead letter from.*Cmd.*")))
   }
 
-  override def beforeEach(): Unit =
+  override def beforeEach(): Unit = {
+    super.beforeEach()
```

Comment:
This is not vital to the test functioning, but it is the only subclass that didn't set up its persistence id properly by calling `super.beforeEach`